spec/requests/set_rights_spec.rb

```diff
@@ -58,175 +16,278 @@
+        it 'sets the access and propagates changes to content metadata' do
+          post "/items/#{pid}/set_rights", params: { access_form: { rights: 'dark' } }
+          expect(response).to redirect_to(solr_document_path(pid))
+          expect(object_client).to have_received(:update)
+            .with(params: updated_model)
+          expect(Argo::Indexer).to have_received(:reindex_pid_remotely).with(pid)
+        end
       end
-    end
 
-    context 'when setting an object to stanford-only' do
-      let(:updated_model) do
-        cocina_model.new(
-          {
-            'access' => {
-              'access' => 'stanford',
-              'download' => 'stanford',
-              'controlledDigitalLending' => false,
-              'embargo' => {
-                'releaseDate' => '2021-02-11T00:00:00.000+00:00',
-                'access' => 'world'
+      context 'when setting stanford-only access' do
+        let(:updated_model) do
+          cocina_model.new(
+            {
+              'access' => {
+                'access' => 'stanford',
+                'download' => 'stanford',
+                'controlledDigitalLending' => false,
+                'embargo' => {
+                  'releaseDate' => '2021-02-11T00:00:00.000+00:00',
+                  'access' => 'world'
+                }
               }
             }
-          }
-        )
-      end
+          )
+        end
 
-      it 'sets the access and does not change content metadata' do
-        post "/items/#{pid}/set_rights", params: { access_form: { rights: 'stanford' } }
-        expect(response).to redirect_to(solr_document_path(pid))
-        expect(object_client).to have_received(:update)
-          .with(params: updated_model)
-        expect(Argo::Indexer).to have_received(:reindex_pid_remotely).with(pid)
+        it 'sets the access and does not change content metadata' do
+          post "/items/#{pid}/set_rights", params: { access_form: { rights: 'stanford' } }
+          expect(response).to redirect_to(solr_document_path(pid))
+          expect(object_client).to have_received(:update)
+            .with(params: updated_model)
+          expect(Argo::Indexer).to have_received(:reindex_pid_remotely).with(pid)
+        end
       end
-    end
 
-    context 'when setting to cdl' do
-      let(:updated_model) do
-        cocina_model.new(
-          {
-            'access' => {
-              'access' => 'citation-only',
-              'download' => 'none',
-              'controlledDigitalLending' => true,
-              'embargo' => {
-                'releaseDate' => '2021-02-11T00:00:00.000+00:00',
-                'access' => 'world'
+      context 'when setting cdl access' do
+        let(:updated_model) do
+          cocina_model.new(
+            {
+              'access' => {
+                'access' => 'citation-only',
+                'download' => 'none',
+                'controlledDigitalLending' => true,
```

Comment:
So access == 'citation-only' and controlledDigitalLending == true is how we model this here, 
but access == 'controlled digital lending' and controlled_digital_lending == true is how we interpret this coming from fedora ?

As shown in this test here...
https://github.com/sul-dlss/dor-services-app/blob/master/spec/services/cocina/from_fedora/dro_access_spec.rb#L109

Rights as "controlled digital lending" is coming from dor-services gem here:

https://github.com/sul-dlss/dor-services/blob/master/lib/dor/datastreams/rights_metadata_ds.rb#L165-L177

via dor-services-app here:

https://github.com/sul-dlss/dor-services-app/blob/master/app/services/cocina/from_fedora/access.rb#L45-L53

Reply:
We made everything consistent in dor-services app... we now say access is 'citation-only' there.